recipes/augeas/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "augeas" %}
+{% set version = "1.8.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "515ce904138d99ff51d45ba7ed0d809bdee6c42d3bc538c8c820e010392d4cc5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://download.augeas.net/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - libxml2
+    - readline
+    - icu
```

Comment:
Please pin according to the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/346dd29465bd97f26baa5427abe29d3476ff9ffa/scripts/pin_the_slow_way.py#L32-L73 ). Also aren't these needed at run time too?